Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -95,6 +95,12 @@
   parent: BaseBullet
   noSpawn: true
   components:
+  - type: Sprite
+    noRot: false
+    sprite: Objects/Weapons/Guns/Projectiles/projectiles2.rsi
+    layers:
+    - state: practice
+      shader: unshaded  
```

```suggestion
      shader: unshaded
```

Please remove unnecessary trailing whitespace.